Makefile

```diff
@@ -1,49 +1,31 @@
 #SHELL := /bin/bash
 
-all: install_deps compile
+all: install
 
 clean:
-	rm -rf build/
 	rm -f $(GOPATH)/bin/todd-server
 	rm -f $(GOPATH)/bin/todd
 	rm -f $(GOPATH)/bin/todd-agent
 
-build: compile
+build:
 	docker build -t mierdin/todd -f Dockerfile .
-	rm -rf build/
 
-compile: install_deps clean
-	mkdir -p build/
-	mkdir -p assets/
-	$(GOPATH)/bin/go-bindata -o assets/assets_unpack.go -pkg="assets" agent/...
-	cd server go build -o ../build/todd-server
-	cd client go build -o ../build/todd
-	cd agent go build -o ../build/todd-agent
+install:
+	go install ./cmd/...
 
 fmt:
 	go fmt github.com/mierdin/todd/...
 
-configureenv:
-	mkdir -p /etc/todd
-	cp -f etc/* /etc/todd/
-	mkdir -p /opt/todd/agent/assets/factcollectors
```

Comment:
Re my follow-up comment, I think this should go back in until we figure out the packaging question.